wiki/Game_Modes/osu!/fr.md

```diff
@@ -0,0 +1,390 @@
+
+Si le bouton de la souris/du clavier a été relâché (ou que le stylo n’a pas touché la tablette) trop tôt pendant la rotation, le spinner arrêtera de prendre en compte la rotation.
+
+Sur la _DS_, c’est un bon moyen de rayer l’écran tactile (ou le protecteur d’écran), en particulier dans les situations difficiles.
+
+Dans osu!, les spinners ont été adoucis pour ne plus être aussi gênants à terminer.
+Le nombre maximum de tours par minute pouvant être atteint est de 477, ce qui correspond également à la vitesse auquel le mod (ou modificateur de jeu) [Auto][Auto wikilink] fait tourner les spinners, tandis que le modificateur de jeu [Spun Out][Spun Out wikilink] les fait tourner plus lentement, à 287 tours par minute.
+
+## Styles de jeu
+
+**[Reportez-vous à la page Styles de jeu dans Osu!.][Play_Styles#Osu! wikilink]**
+
+## Contrôles
+
+Les contrôles par défaut pour osu! sont:
+
+Souris|Clavier|Tablette/Écran tactile
+---|---|---|
+Click gauche(M1) / Clic-droit(M2)|`Z`(K1) / `X`(K2)|Toucher l'écran(M1)
+
+Les hit objects (ou objets cliquables) dans osu! accepteront tous types d'entrées du périphérique d'entrée, à condition que le hit object touché ait été frappé à temps.
+
+Si le Mod [Relax][Relax wikilink] a été utilisé, seul le curseur fonctionnera en jeu.
+Utilisez le curseur du jeu pour suivre les hit objects avec les frappes automatisé.
+Les spinners doivent encore être complétés.
+
+Si le Mod [Auto Pilot][Auto Pilot wikilink] a été utilisé, seule les frappes du périphérique d’entrée fonctionneront.
+Respectez le timing des hit objects avec le mouvement automatique du curseur.
+Les spinners suivront la vitesse du mod [Spun Out][Spun Out wikilink]
+
+## Calcul du score
+
+_[Les valeurs de base peuvent être trouvées dans la section Score dans Osu! Section des Scoring Values][Score#Osu!SV wikilink]_
+
+La section score détaille toutes les subtilités du score, y compris la formule mathématique.
+
+### Notes
+
+Note| Condition
+:---:|:---
+SS | 100% de précision
+S  | Plus de 90% 300, moins que 1% de 50 et aucun miss
+A  | Plus de 80% 300 et aucun miss **OU** plus de 90% de 300
+B  | Plus de 70% de 300 et aucun miss **OU** plus de 80% de 300
+C  | Plus de 60% de 300
+D  | Le reste
+
+Un S ou SS en argent ne peut être obtenu qu'en utilisant le mod _[Hidden][Hidden wikilink]_ et/ou le mod _[Flashlight][Flashlight wikilink]_.
+
+### Précision (ou Accuracy)
+
+La précision est calculée en additionant le nombre de points obtenu sur chaque note et en le divisant par le montant maximal possible.
+
+En d'autres termes: `Précision = Total de points de hits / (Nombre total de hits * 300p)`
+
+Terme | Formule
+:---:|:---
+**Total de points de hits**   | (50s * 50p) + (100s * 100p) + (300s * 300p)
+**Nombre total de hits**   | (Misses + 50s + 100s + 300s) * 300
+**s**                      | Nombre de
```

Comment:
@cl8n is it a mistake? i've checked the en.md and it also occur in there, i'm not sure if we should keep it as it doesn't really make sense...